schemas/hygiene/aep-hygiene-ops-record.schema.json

```diff
@@ -5,12 +5,11 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/aep-hygiene-ops",
+  "$id": "https://ns.adobe.com/xdm/schemas/aep-hygiene-ops",
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "AEP Hygiene Operation Request",
   "description": "AEP Data Hygiene operations that instruct AEP to delete or modify records in a specified dataset and sandbox.",
   "type": "object",
-  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
   "meta:extensible": false,
   "meta:abstract": false,
   "definitions": {
```

Comment:
Need to add a new field "xdm:workorderID" to the schama as follows:
`        "xdm:workorderID": {
          "title": "Work Order ID for tracking purposes.",
          "description": "Each row in a batch will be produced by a different customer REST API request, and will therefore have a different workorderID for tracking purposes.",
          "type": "string"
        }`